### DISCUSSION TITLE
Future

### DISCUSSION THREAD

**Discussion by zamkorus** [rvaidun/berealviewer#138] [General]
@rvaidun In view of the situation, are you planning any development of what's left here on github, or are you abandoning the project, and will it just run as long as it goes.

**Comment by rvaidun** [comment]
I will continue to develop the project as long as I am still interested in it. I am a big user of the site myself.

**Reply by zamkorus** [reply]
@rvaidun situation changed?

**Reply by rvaidun** [reply]
no site went up accidentally briefly since it automatically deploys main branch on every update

**Comment by rvaidun** [comment]
site is back

**Reply by Ein-Tim** [reply]
Did something change about the legal situation?
